script/run-container.sh

```diff
@@ -31,6 +31,7 @@ TNF_OMIT_ARTIFACTS_ZIP_FILE_DEFAULT=false
 TNF_INCLUDE_WEB_FILES_IN_OUTPUT_FOLDER_DEFAULT=false
 ON_DEMAND_DEBUG_PODS_DEFAULT=false
 TNF_ENABLE_DATA_COLLECTION_DEFAULT=false
+TNF_ENABLE_XML_CREATION=false
```

Comment:
Shouldn't this be TNF_ENABLE_XML_CREATION_DEFAULT? with a default of true, since the old file was always generated before for DCI